terraform/aws/addons/main.tf

```diff
@@ -75,3 +82,82 @@ resource "null_resource" "nginx_ingress_install" {
     command = "kubectl delete --kubeconfig ${var.config_path} -f 'https://raw.githubusercontent.com/shalb/terraform-aws-minikube/master/addons/ingress.yaml'"
```

Comment:
Hmm, not pinned version.

We should grab release version from pipeline, then send to TF as `var` and use it.

Because, in master this file can be moved/deleted in future